src/Infrastructure/LeanCode.OpenTelemetry/IdentityTraceBaggageHelpers.cs

```diff
@@ -6,17 +6,19 @@ namespace LeanCode.OpenTelemetry;
 public static class IdentityTraceBaggageHelpers
 {
     public const string UserIdKey = "enduser.id";
-    public const string RoleKey = "enduser.role";
+    public const string UserRoleKey = "enduser.role";
```

Comment:
Yeah, as Emil said, we probably should use `enduser` as "initiator" (but let's not rename them - let's keep `EndUser` name) and something like `current_user` for `User`.